### ISSUE TITLE
Enable force update option for stream update

### ISSUE THREAD

**Issue by ilayaperumalg** [spring-cloud/spring-cloud-dataflow-ui#1504]
It might be handy to have `stream update` forced. Currently, the update can only happen when there are properties difference between current and the previous deployment.

**Comment by jvalkeal** [comment]
@ilayaperumalg But wouldn't this cause unnecessary update to an apps which have not changed? Like if I update `time` on `time|log` why I'd wan't to touch running `log`. Though, there have been times I've liked to know what update would actually do before actually doing it.

**Comment by ilayaperumalg** [comment]
> But wouldn't this cause unnecessary update to an apps which have not changed? Like if I update time on time|log why I'd wan't to touch running log. Though, there have been times I've liked to know what update would actually do before actually doing it.

Yes, it will. But, there are cases where one would want to restart their apps (ex: a restart might be required to sync up with some external change) and in those cases the only option available is to undeploy/re-deploy. But this option has its own disadvantage as the deployment properties need to be provided again. 

In a way, this is also related to https://github.com/spring-cloud/spring-cloud-dataflow/issues/4058 where we can re-use the deployment properties for subsequent deployments (from the Skipper deployment manifest).
